conf/etc/v2/ha.conf

```diff
@@ -0,0 +1,5 @@
+LOG:
```

Comment:
can we have 
```
VERSION:
    version: <VERSION> 
```
This in old ha.conf also

Reply:
Done